Xamarin.Forms.Platform.iOS/Renderers/EntryRenderer.cs

```diff
@@ -116,11 +116,12 @@ protected override void OnElementChanged(ElementChangedEventArgs<Entry> e)
 			_cursorPositionChangePending = Element.IsSet(Entry.CursorPositionProperty);
 			_selectionLengthChangePending = Element.IsSet(Entry.SelectionLengthProperty);
 
+			// Font needs to be set before Text and Placeholder so that they layout correctly when set
```

Comment:
this is the only change that could in theory cause breaking change to existing behavior